frontend/src/pages/SubProjects/ProjectHistoryDrawer.js

```diff
@@ -2,29 +2,28 @@ import React from "react";
 import { connect } from "react-redux";
 import HistoryDrawer from "../Common/History/HistoryDrawer";
 import { hideHistory } from "../Notifications/actions";
-import { fetchNextProjectHistory } from "./actions";
+import { fetchNextProjectHistoryPage } from "./actions";
 
 function ProjectHistoryDrawer({
   projectId,
-  offset,
-  limit,
   doShow,
   events,
   nEventsTotal,
-  hasMore,
+  currentHistoryPage,
+  lastHistoryPage,
   isLoading,
   getUserDisplayname,
   hideHistory,
-  fetchNextProjectHistory
+  fetchNextProjectHistoryPage
 }) {
   return (
     <HistoryDrawer
       doShow={doShow}
       onClose={hideHistory}
       events={events}
       nEventsTotal={nEventsTotal}
-      fetchNext={() => fetchNextProjectHistory(projectId, offset, limit)}
-      hasMore={hasMore}
+      fetchNext={() => fetchNextProjectHistoryPage(projectId)}
+      hasMore={currentHistoryPage !== lastHistoryPage}
```

Comment:
```suggestion
      hasMore={currentHistoryPage <== lastHistoryPage}
```

Reply:
done

Reply:
has to be `hasMore={currentHistoryPage < lastHistoryPage}`